molecule/cloud-epc-delegated/verify.yml

```diff
@@ -0,0 +1,16 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
```

Comment:
duplicate molecule/resources/tests/verify.yml

Reply:
It's not duplicate. It's a workaround to avoid failures on ansible 2.8 runners in EPC.